### PR TITLE
PC update

### LINKED ISSUE #48: Describe "suggest features" in pixel classification docs

### PR DESCRIPTION
**Pull request by akreshuk**
Update descriptions and screenshots, add a bit about the "Suggest Features" button. Fixes #48.

### CODE REVIEW

File: documentation/pixelclassification/pixelclassification.md [hunk 1]
```diff
@@ -35,56 +38,58 @@ Nice properties of the algorithm and workflow are
...
+After the data is loaded, switch to the next applet **Feature Selection**.
+Here you will select the pixel features and their scales which in the next step will be used to discriminate between the different classes of pixels. 
+
 A click on the **Select features** button brings up a feature selection dialog.
 
-<a href="snapshots/feature_selection_zoomed.png" data-toggle="lightbox"><img src="snapshots/feature_selection_zoomed.png" class="img-responsive" /></a>
-<a href="snapshots/feature_selection2.png" data-toggle="lightbox"><img src="snapshots/feature_selection2.png" class="img-responsive" /></a>
+<a href="snapshots/feature_selection_add.png" data-toggle="lightbox"><img src="snapshots/feature_selection_add.png" class="img-responsive" /></a>
+
+For 3D data the features can be computed either in 2D or 3D. 2D can be useful if data has thick slices and the information from a slice is not so relevant for the neighbors. It is also the only way to compute large-scale filters in think stacks. The following image shows the switch between 2D and 3D computation in the Feature Selection dialog.
```
Comment (Tomaz-Vieira) [inline documentation/pixelclassification/pixelclassification.md]: I'm guessing it should be "thin stacks" ?

Comment (akreshuk) [reply documentation/pixelclassification/pixelclassification.md]: Yes, thanks :)

Comment (akreshuk) [reply documentation/pixelclassification/pixelclassification.md]:
```suggestion
For 3D data the features can be computed either in 2D or 3D. 2D can be useful if data has thick slices and the information from a slice is not so relevant for the neighbors. It is also the only way to compute large-scale filters in thin stacks. The following image shows the switch between 2D and 3D computation in the Feature Selection dialog.
```

File: documentation/pixelclassification/pixelclassification.md [hunk 2]
```diff
@@ -97,6 +102,21 @@ To access the "Import Labels" feature in the GUI, do the following:
...
+
+
+It works by evaluating the classifier predictions using different feature sets on the pixels you have already annotated. The so called "out-of-bag" predictions are used, so for every tree in the Random Forest we only use the pixels it has not seen in training to estimate the error. Still, even though the pixels were not seen in training, randomization is done on the pixel level, so the tree has likely seen some similar pixels and the error estimate is overly optimistic. Besides the error, ilastik also reports the computing time which can be helpful when trying to find the balance between runtime and accuracy. By choosing the feature set in the lower left corner, you can also see the predictions ilastik would make with these features on the current field of view.
+
+<a href="snapshots/suggest_features_results_2.png" data-toggle="lightbox"><img src="snapshots/suggest_features_results_2.png" class="img-responsive" /></a>
+
+If you like one of the suggested features sets better than your current selection, you can replace your selection with the a new set by pressing the **Select Feature Set** button.
```
Comment (Tomaz-Vieira) [inline documentation/pixelclassification/pixelclassification.md]: there's a loose extra "the" in "(...) with the a new set"

Comment (akreshuk) [reply documentation/pixelclassification/pixelclassification.md]:
```suggestion
If you like one of the suggested features sets better than your current selection, you can replace your selection with the new set by pressing the **Select Feature Set** button. 
```

**Comment by k-dominik** [comment]
In the end I preferred your wordings in overlapping passages and just added two screenshots that I have redone:

Teaser image:
b1

Window leveling snap:
b2

**Pull request merged**